software/base/src/main/java/brooklyn/entity/software/MachineLifecycleEffectorTasks.java

```diff
@@ -561,7 +560,7 @@ public void stop(ConfigBag parameters) {
 
         boolean isEntityStopped = entity().getAttribute(SoftwareProcess.SERVICE_STATE_ACTUAL)==Lifecycle.STOPPED;
 
-        DynamicTasks.queue("pre-stop", new Callable<String>() { public String call() {
+        DynamicTasks.queue("pre-stop of " + entity() + ", submitted by " + Tasks.current(), new Callable<String>() { public String call() {
```

Comment:
It's just a discrepancy between the code that ran to in the Clocker PR and the code listed above. Since it's contentious I'll just drop it from this pull request.